### DISCUSSION TITLE
Lost Tokens

### DISCUSSION THREAD

**Discussion by gator6615** [WalletConnect/walletconnect-monorepo#1038] [Developer Support]
I'm in desperate need of some expert help I guess? I need to get my Uniswap tokens to a usable wallet I was told you guys here at wallet connect were the experts! I have all the transactional data on a PDF doc. I've been Scammed alot in the past, so I really dont know what or where to go next. I've tried contacting Uniswap and the wallet used MetaMask but with ZERO success. I'm a single parent struggling so these tokens were a god send if I could only get to them?

Thanks

**Comment by chadyj** [comment]
Sorry to hear that you got scammed.

We can't help you get your tokens back, but we do have some safety tips to keep in mind https://walletconnect.com/faq#safety
